source/CPluginSystem.cpp

```diff
@@ -25,9 +26,12 @@ void CPluginSystem::LoadPlugins()
 
         for (DWORD i = 0; i < files.count; i++)
         {
-            if (std::find(filenames.begin(), filenames.end(), files.strings[i]) != filenames.end())
+            if (std::find(paths.begin(), paths.end(), files.strings[i]) != paths.end())
                 continue; // file already listed
 
+            if (skippedPaths.find(files.strings[i]) != skippedPaths.end())
+                continue; // file already skipped
```

Comment:
how it is possible to meet this condition?

we scan 
```
CLEO\cleo_plugins\SA.*.cleo
CLEO\cleo_plugins\*.cleo
CLEO\*.cleo
```
so filenames are unique across all three scans